addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -670,42 +670,42 @@
         <we-row string="Links" class="o_design_tab_title">
             <we-colorpicker data-customize-css-property=""
                             data-css-property="color"
-                            data-selector-text="a:not(.btn), a.btn-link"
+                            data-selector-text="a:not(.btn), a.btn.btn-link"
```

Comment:
No these are to create css styles with this selector text. `data-selector-text` was created specially for that purpose.
It just so happens that when you want to get the style matching that selector to show the values in the sidebar, you basically have two options :
1. Go fetch the values in the design-element stylesheet
    This is problematic because then you have to write the exact same selector to find it, and if it's not defined then you got nothing. That means that if a theme wants to declare some global styles, it has to re-write _all of them_. This is what we actually used to do until we switched to option 2.
2. Create an element matching the selector and add it to the dom so you can getComputedStyle on it
    This means that if you want to be able to get the actual style in the DOM of the element, your selector has to be precise enough that it captures the styles you need to capture. That's why here we have to add .btn because we were missing some default styles that were declared by bootstrap.
Yes the whole thing is a bit of a Frankenstein monster, that's because we're juggling between styles passed by Odoo/Bootstrap via stylesheets, styles declared via the design tab, and styles declared locally via snippet options and editor toolbar.

Reply:
Alright then I misunderstood what `data-selector-text` was supposed to mean.